inc/API.php

```diff
@@ -92,8 +95,24 @@ public static function peerboard_api_call($slug, $token = 0, $body, $type = 'GET
 
     if ($check_success) {
       $success = self::check_request_success($request, func_get_args());
-
-      if (!$success) {
+      // If we have issue with ssl
+      if ($success === 'ssl_fix') {
+        // Try to update certificate and fix ssl issue 
+        $ssl_fix = self::update_wp_ca_bundle();
+
+        if (isset($ssl_fix['success'])) {
+          // Make the same request
+          $success = self::peerboard_api_call($slug, $token, $body, $type, $api_url);
```

Comment:
I think we should check $success === 'ssl_fix' here again.

Reply:
Or we can retry some request that shouldn't be retried

Reply:
Ready